apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -69,53 +75,32 @@ public void Dispose()
             await FillSampleData();
         }
 
-        private async Task<Database> CreateDatabaseAsync()
+        private async Task CreateDatabaseAsync()
         {
-            // Create client
-            _databaseAdminClient = await DatabaseAdminClient.CreateAsync();
-            // Initialize request argument(s)
-            var parent = new InstanceName(TestProjectName, TestInstanceName);
-
-            string createStatement = "CREATE DATABASE " + DatabaseName;
-
-            // Make the request
-            var response = await _databaseAdminClient.CreateDatabaseAsync(parent, createStatement);
-
-            // Poll until the returned long-running operation is complete
-            var completedResponse = await response.PollUntilCompletedAsync();
-
-            // Retrieve the operation result
-            return completedResponse.Result;
+            using (var connection = new SpannerConnection(NoDbConnectionString))
+            {
+                var createCmd = connection.CreateDdlCommand("CREATE DATABASE " + DatabaseName);
+                await createCmd.ExecuteNonQueryAsync(CancellationToken.None);
```

Comment:
You shouldn't need to specify CancellationToken.None here - the DbCommand implementation will pass that to the SpannerCommand implementation, won't it?